docs/core/docker/build-container.md

```diff
@@ -283,7 +283,18 @@ Save the *Dockerfile* file. The directory structure of the working folder should
 
 :::zone-end
 
-From your terminal, run the following command:
+The `ENTRYPOINT` instruction sets `dotnet` as the host for the `DotNet.Docker.dll`. However, it's possible to instead define the `ENTRYPOINT` as the app executable itself, relying on the OS as the app host:
+
+```dockerfile
+ENTRYPOINT ["./DotNet.Docker"]
+```
+
+This causes the app to be executed directly, without `dotnet`, and instead relies on the app host and the underlying OS. There should be no app behavior difference when launching from via app host versus the `dotnet` host for ordinary server apps. Launching via an app host gives you better integration with the underlying OS. For example:
+
+- You'll see the app host name in your process list and not `dotnet`, which could be confusing if there is more than one.
+- You can customize the app host with OS specific features. For more information, see [this discussion about configuring default stack size on Windows](https://github.com/dotnet/runtime/issues/96347#issuecomment-1981470713).
+
+To build the container, from your terminal, run the following command:
```

Comment:
I think the detailed discussion of the advantages / disadvantages should be moved near https://learn.microsoft.com/en-us/dotnet/core/deploying/#produce-a-cross-platform-binary and this doc should just link to it.

Reply:
I can see your point, but maybe I should link to the article you reference here instead. I feel as though it's best to keep this information here, as it directly relates to multiple customer concerns. What do you think about adding a link from this doc to the doc you mentioned?

Reply:
The detailed discussion of the `dotnet` vs. app host differences is advanced topic, and it would be best to keep it to minimum in this tutorial doc. I believe that it is irrelevant content to 90+% of people going through this tutorial.

Reply:
Fair enough, I'll update this PR.